packages/DOS/common.dj

```diff
@@ -91,33 +104,59 @@ USE_ARES = 0
 #
 # Enable debug code in libcurl/curl
 #
-USE_DEBUG = 0
+USE_DEBUG = 1
 
 #
 # Enable memory tracking code in libcurl/curl
 #
 USE_CURLDEBUG = 0
 
+#
+# Generate a .map file in 'link_EXE' macro
+#
+MAKE_MAP_FILE ?= 1
+
 default: all
 
 #
 # Root directory for Waterloo tcp/ip etc. Change to suite.
 # WATT_ROOT should be set during Watt-32 install.
 #
-WATT32_ROOT  = $(subst \,/,$(WATT_ROOT))
-OPENSSL_ROOT = e:/net/openssl.099
-ZLIB_ROOT    = $(DJDIR)/contrib/zlib
-LIBIDN_ROOT  = $(TOPDIR)/../IDN/libidn
-ARES_ROOT    = $(TOPDIR)/ares
-
-CC   = gcc
-YACC = bison -y
+WATT32_ROOT   = $(realpath $(WATT_ROOT))
+OPENSSL_ROOT ?= $(TOPDIR)/../crypto/OpenSSL
+ZLIB_ROOT    ?= e:/djgpp/contrib/zlib
+LIBIDN_ROOT  ?= $(TOPDIR)/../IDN/libidn
+ARES_ROOT    ?= $(TOPDIR)/../DNS/c-ares
 
 CFLAGS = -g -O2 -I. -I$(TOPDIR)/include -I$(TOPDIR)/lib \
          -I$(WATT32_ROOT)/inc -Wall -DHAVE_CONFIG_H
 
-ifeq ($(USE_SSL),1)
-  CFLAGS += -DUSE_OPENSSL -I$(OPENSSL_ROOT)
+ifeq ($(USE_OPENSSL),1)
+  CFLAGS += -DUSE_OPENSSL -I$(OPENSSL_ROOT)/include
+
+  #
+  # Squelch the warnings from '$(OPENSSL_ROOT)/include/openssl/opensslconf.h'
+  # and use of deprecated functions.
+  #
+  CFLAGS += -D__AFL_LOOP -DOPENSSL_SUPPRESS_DEPRECATED
```

Comment:
Why is __AFL_LOOP defined here

Reply:
Something to do with an unused `static int __afl_lop()` function in a OpenSSL header; AFL fuzzy code.

Reply:
> Something to do with an unused `static int __afl_lop()` function in a OpenSSL header;

I've never heard of this, I think it's specific to your build. Are you using a fuzzer?

Reply:
> Are you using a fuzzer?

No. I'm not sure how the fuzzing is OpenSSL is  supposed to work.
But removing `-D__AFL_LOOP` just now and rebuilding, the previous warnings are no longer present. 
Hence I drop this define.